### ISSUE TITLE
Basic-carousel: height is 0

### ISSUE THREAD

**Issue by Jamtis** [basic-web-components/basic-web-components#8]
I recognized that the `basic-carousel` has no height in a certain case.

https://jsfiddle.net/jamtis/m5sf1zp9/

I copied the code from https://github.com/JanMiksovsky/carousel-with-tabs/blob/master/index.html.

Maybe I overlooked something...

It seems that `basic-carousel` does not respond to `flex: 1;`.
Same behavior in both shadow and shady DOM.

I think this is a quite strange behavior...

**Comment by JanMiksovsky** [comment]
Your example is missing a single, but important, line of CSS from the demo: the `display: flex` directive:

```
basic-arrow-direction, basic-page-dots, basic-carousel {
    display: flex;
    flex: 1;
}
```

Without `display: flex` on the container, the `flex: 1` on the children won't work. Adding the missing line back restores the carousel layout: https://jsfiddle.net/Jan_Miksovsky/5fv5td9q/.

Please confirm that this works for you.

**Comment by Jamtis** [comment]
Thanks, that's works.

However it's confusing because chrome dev tools show:

```
:host {
  display: flex:
}
```

So I didn't consider adding an explicit `display: flex;`.

I updated the fiddle, so that it shows the `display` property of basic-carousel without the explicit CSS.

https://jsfiddle.net/jamtis/m5sf1zp9/3/

**Comment by JanMiksovsky** [comment]
Ah, good catch. It turns out the problem was with basic-page-dots, not basic-carousel. Unlike the other components in this carousel family, basic-page-dots didn't use flex or inline-flex on its host. I've checked in a fix for that, which will go out in the 0.6.4 release. In the meantime, you can set display: inline-flex / -webkit-inline-flex on basic-page-dots.

You'll still need to set flex: 1 on these components.

**Comment by JanMiksovsky** [comment]
This has now been released in v0.6.4.
